packages/theme/src/react/__specs__/with-default-theme.spec.js

```diff
@@ -1,11 +1,11 @@
 import { mount } from 'enzyme'
```

Comment:
Might be reading the diff wrong, but might want to rename the file too.

Reply:
you're right!